### ISSUE TITLE
Can I use the left button to select an area on Linux, just like bandicam?

### ISSUE THREAD

**Issue by flowerseacar** [vkohaupt/vokoscreenNG#319]

**Comment by vkohaupt** [comment]
No.

**Comment by flowerseacar** [comment]
> No.

Can this function be optimized?

**Comment by vkohaupt** [comment]
No

**Comment by flowerseacar** [comment]
> No

Don't really want to talk

**Comment by vkohaupt** [comment]
Correct. I'm not going to install Bandicam and then tell you, piece by piece, everything about what the button is about.
Please make a feature request and describe it step by step.

**Comment by Templayer** [comment]
> Correct. I'm not going to install Bandicam and then tell you, piece by piece, everything about what the button is about. Please make a feature request and describe it step by step.

I think he means this: https://www.youtube.com/watch?v=tUrgx0-kZ7s
